sql-cli/sql_cli/astro/command.py

```diff
@@ -0,0 +1,18 @@
+import os
+
+import click
+import typer
+from typer.core import TyperCommand
+
+
+class AstroCommand(TyperCommand):
+    def format_usage(self, ctx: typer.Context, formatter: click.HelpFormatter) -> None:
+        """Writes the usage line into the formatter.
+
+        This is a low-level method called by :meth:`get_usage`.
+        """
+        pieces = self.collect_usage_pieces(ctx)
+        command_path = ctx.command_path
+        if os.getenv("ASTRO_CLI"):
```

Comment:
Perhaps we can add a comment here that this environment variable is set in the astro binary. Or we can reference https://github.com/astronomer/astro-cli/pull/855 somewhere? WDYT?

Reply:
Makes sense 👍